templates/case/communication/digital_post/index.html.twig

```diff
@@ -0,0 +1,46 @@
+{% extends 'layout-with-case-submenu.html.twig' %}
+
+{% trans_default_domain 'digital_post' %}
+
+{% block title %}{% trans %}Digital post{% endtrans %}{% endblock %}
+
+{% block header %}
+    <div class="row">
+        <div class="col">
+            <h1>{% trans %}Digital post{% endtrans %}</h1>
+        </div>
+    </div>
+{% endblock %}
+
+{% block content %}
+    <table class="table table-striped">
+        <thead>
+            <tr>
+                <th>{% trans %}Status{% endtrans %}</th>
+                <th>{% trans %}Created at{% endtrans %}</th>
+                <th>{% trans %}Sent at{% endtrans %}</th>
+                <th>{% trans %}Document{% endtrans %}</th>
+                <th class="number">{% trans %}Number of attachments{% endtrans %}</th>
+                <th></th>
```

Comment:
It occurred to me that we don't show recipients anywhere. Consider adding it here or at least in `digital_post_show`.

Reply:
I've added it to the details view.